1237-registrator_worklist/1249-creator_adds_a_new_message_on_a_doi_request.feature

```diff
@@ -2,9 +2,9 @@ Feature: Creator adds a new message on a DOI request
 
     @1249
     Scenario: Creator adds a new message on a DOI request
-        Given that the Creator Opens a DOI request entry from My Worklist
-        And the request is Answered
-        And they see the answer from the Curator
-        When the they click the New Message button
-        Then they can add a new message to the DOI request
-        And the request changes status to Requested
\ No newline at end of file
+        Given that the Creator views details of a DoiRequest in the Messages list
+        And the request has an answer from a Curator
+        And they see the message from the Curator
+        When the Creator enters a new message
+        And clicks the Send Answer button
+        Then they can see that the new message is added to the list of messages
```

Comment:
```suggestion
        Then they see that the new message is added to the list of messages
```

Den eneste gangen det er referert til en list of messages er implisitt i Given'en, så jeg tror det er bedre å skrive ut Given'en slik at det blir tydelig at det finnes en liste av meldinger i starten også